src/wsgi.py

```diff
@@ -1,8 +1,7 @@
 import os
 
 from django.core.wsgi import get_wsgi_application
-from whitenoise.django import DjangoWhiteNoise
 
-settings_module = os.environ.setdefault('DJANGO_SETTINGS_MODULE', "settings.base")
+settings_module = os.environ.setdefault('DJANGO_SETTINGS_MODULE', "settings.develop")
 
-application = DjangoWhiteNoise(get_wsgi_application())
+application = get_wsgi_application()
```

Comment:
Where does whitenoise stuff get declared now? Did I miss it?

Reply:
Whitenoise is used in development only, via:

```python
# settings/develop.py

MIDDLEWARE += ('whitenoise.middleware.WhiteNoiseMiddleware',)
```

Reply:
yea, but don't we still have to wrap our wsgi app in whitenoise stuff? not just activate the middleware?

Reply:
http://whitenoise.evans.io/en/stable/django.html#enable-whitenoise

No I don't think we need to do WSGI stuff for Django, just middleware is fine.